apps/documentation/src/routes/documentation/tutorial/components.mdx

```diff
@@ -1,12 +1,10 @@
-import { Head } from 'tuono'
-
-<Head>
-  <title>Tutorial - Components</title>
-  <link
-    rel="canonical"
-    href="https://tuono.dev/documentation/tutorial/components"
-  />
-</Head>
+import MetaTags from '../../../components/meta-tags'
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>This sentence does not start with an uppercase letter. (UPPERCASE_SENTENCE_START)
Suggestions: `Import`
URL: https://languagetool.org/insights/post/spelling-capital-letters/ 
Rule: https://community.languagetool.org/rule/show/UPPERCASE_SENTENCE_START?lang=en-US
Category: CASING
<!-- __reviewdog__:ChBhNTM2NjIwODU4N2QyNWYwEgxMYW5ndWFnZVRvb2w= -->